sql/parser/normalize_test.go

```diff
@@ -72,6 +72,7 @@ func TestNormalizeExpr(t *testing.T) {
 		{`1=a/2`, `a = 2`},
 		{`a=lower('FOO')`, `a = 'foo'`},
 		{`lower(a)='foo'`, `lower(a) = 'foo'`},
+		{`random()`, `random()`},
```

Comment:
I'd really like to see a test in `testdata/order_by` and perhaps one in `testdata/builtin` as well. This is tricky because `random()` is not deterministic. But I think we should fix that by allowing an optional seed argument to `random()`. File an issue if you don't want to do this in this PR.